crowbar_framework/config/application.rb

```diff
@@ -63,5 +63,7 @@ class Application < Rails::Application
         Rails.logger.warn "Failed to load chef"
       end
     end
+    # experimental options
+    config.experimental = config_for(:experimental)
```

Comment:
How does the Crowbar admin enable this option?  I know this is only a POC currently but it would be nice if this was available via the UI / API.

Reply:
Discussed on IRC - initially it's a `config/experimental.yml` file with toggles for each type of experimental option, but later we can expose it via the UI / API.